Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlSettings.cs

```diff
@@ -0,0 +1,80 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Security;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
+{
+    /// <summary>
+    /// A proxy for <see cref="Windows.Web.UI.WebViewControlSettings"/>. This class cannot be inherited.
+    /// </summary>
+    /// <seealso cref="Windows.Web.UI.WebViewControlSettings"/>
+    public sealed class WebViewControlSettings
+    {
+        [SecurityCritical]
+        private readonly Windows.Web.UI.WebViewControlSettings _settings;
+
+        internal WebViewControlSettings(Windows.Web.UI.WebViewControlSettings settings)
+        {
+            _settings = settings ?? throw new ArgumentNullException(nameof(settings));
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the use of IndexedDB is allowed.
+        /// </summary>
+        /// <value><c>true</c> if IndexedDB is allowed; otherwise, <c>false</c>. The default is <c>true</c>.</value>
+        // ReSharper disable InconsistentNaming
```

Comment:
Why are there resharper comments in here?

Reply:
Thought there was precedent for this (they're also in `AdvancedCollectionView.cs` and `AdvancedCollectionView.Events.cs` in `Microsoft.Toolkit.Uwp.UI`). I'll remove them